### ISSUE TITLE
You are included the JEI jar in your mod, you should be having it as a dependent

### ISSUE THREAD

**Issue by kgarms** [shedaniel/REIPluginCompatibilities-Issues#50]
### What happened?

You are included the JEI jar in your mod, you should be having it as a dependent, otherwise no one can include the jei for server packs without forcefully telling all the other mods to load even though it appears the jei mod they need is not included.

I will include the important part of the server logs that is causing the issue.

To make the server work I simple have to disable the REIPluginCompatibilities-forge-12.0.55.jar and then server starts perfectly with no issues.

What you need to do is make the jei a dependency of your mod, not have it included in your mod.

### Relevant log output

```shell
[11:24:55] [main/ERROR]: Found duplicate mods:
	Mod ID: 'jei' from mod files: REIPluginCompatibilities-forge-12.0.55.jar, jei-1.20.1-forge-15.2.0.22.jar
[11:24:55] [main/ERROR]: Failed to build unique mod list after mod discovery.
net.minecraftforge.fml.loading.EarlyLoadingException: Duplicate mods found
	at net.minecraftforge.fml.loading.UniqueModListBuilder.buildUniqueList(UniqueModListBuilder.java:87) ~[fmlloader-1.20.1-47.1.0.jar:1.0]
	at net.minecraftforge.fml.loading.moddiscovery.ModDiscoverer.discoverMods(ModDiscoverer.java:106) ~[fmlloader-1.20.1-47.1.0.jar:?]
	at net.minecraftforge.fml.loading.FMLLoader.beginModScan(FMLLoader.java:164) ~[fmlloader-1.20.1-47.1.0.jar:1.0]
	at net.minecraftforge.fml.loading.FMLServiceProvider.beginScanning(FMLServiceProvider.java:86) ~[fmlloader-1.20.1-47.1.0.jar:1.0]
	at cpw.mods.modlauncher.TransformationServiceDecorator.runScan(TransformationServiceDecorator.java:112) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.TransformationServicesHandler.lambda$runScanningTransformationServices$8(TransformationServicesHandler.java:100) ~[modlauncher-10.0.9.jar:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197) ~[?:?]
	at java.util.HashMap$ValueSpliterator.forEachRemaining(HashMap.java:1779) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:575) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluateToArrayNode(AbstractPipeline.java:260) ~[?:?]
	at java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:616) ~[?:?]
	at java.util.stream.ReferencePipeline.toArray(ReferencePipeline.java:622) ~[?:?]
	at java.util.stream.ReferencePipeline.toList(ReferencePipeline.java:627) ~[?:?]
	at cpw.mods.modlauncher.TransformationServicesHandler.runScanningTransformationServices(TransformationServicesHandler.java:102) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.TransformationServicesHandler.initializeTransformationServices(TransformationServicesHandler.java:55) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:88) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?]
[11:24:55] [main/ERROR]: Mod Discovery failed. Skipping dependency discovery.
```

### By submitting this issue, I have included the necessary logs by pasting the contents into the correct location or attaching the file as an upload.

- [X] Yes, and I did not use any paste services other than GitHub Gists.

### By submitting this issue, I have confirmed my REI and REI's dependencies are up to date.

- [X] Yes

**Comment by shedaniel** [comment]
REI Plugin Compatibilities is a reimplementation of the JEI api, it does not include JEI.
This mod is designed to pretend to be JEI, without **being** JEI, we don't want JEI to load, and JEI shouldn't load, because that is the job of REI.

TL;DR Use JEI or REI with REIPC
